cmd/lakefs/cmd/run.go

```diff
@@ -21,7 +21,6 @@ import (
 	"github.com/treeverse/lakefs/auth/crypt"
```

Comment:
can you remove the Catalog print in this file - had no line to report this one.

Reply:
Please keep it for a while -- so users (and us) know what's running.

Reply:
But for this version, there is no point in printing it, right?